python/lib/listitembuilder.py

```diff
@@ -15,6 +21,8 @@
 
 def build_video_listitem(item):
     result = xbmcgui.ListItem(item.get('label'))
+    if 'title' in item:
```

Comment:
Set 'title' in the `infokey_map` along with the other new keys, rather than overwriting label here.